src/main/java/com/lambda/client/mixin/client/gui/MixinGuiChest.java

```diff
@@ -14,12 +14,10 @@
 @Mixin(GuiChest.class)
 public class MixinGuiChest {
 
-    @Final @Shadow private IInventory lowerChestInventory;
-
-    @Inject(method = "drawScreen", at = @At("RETURN"))
-    public void drawScreen(int mouseX, int mouseY, float partialTicks, CallbackInfo ci) {
-        if (lowerChestInventory.getName().equals("Ender Chest")) {
-            ContainerPreview.INSTANCE.setEnderChestItems(lowerChestInventory);
+    @Inject(method = "<init>", at = @At("RETURN"))
```

Comment:
I know that on Draw isn't the best method, however frequently the ender chests inventory doesn't seem to be populated on init. Further this means that changes to your ember chest won't be saved.

Reply:
`lowerChestInventory` has `final` modifier so it won't change after init